### DISCUSSION TITLE
软件坏了

### DISCUSSION THREAD

**Discussion by Mr-Zhang-Cn** [lyswhut/lx-music-desktop#1758] [General]
2024年1月27日 17:46:43   就在这个时间  我使用了软件,检索了周杰伦,随便播放了几首歌 一个出声的都没有  我又找了孙子涵 播了几个 依然GG 进度条那里显示url过期

**Comment by MandragoraWhite** [comment]
更新公告你是一点不看啊 被投诉移除所有内置音源了

**Comment by opser-gavin** [comment]
> 更新公告你是一点不看啊 被投诉移除所有内置音源了

 去掉内置音源是从哪个版本开始的 找了下没找到呢

**Comment by turbo2288** [comment]
小心看此链接 ： https://v.douyin.com/iNC3v7XX/ 
要用2.5.0版本！
